independent-publisher-connectors/Ron Swanson Quotes/readme.md

```diff
@@ -0,0 +1,21 @@
+# Ron Swanson Quotes
+Quotes for every occasion. Because Ron Swanson is a hero.
+
+## Publisher: Troy Taylor, Hitachi Solutions
+
+## Prerequisites
+There are no prerequisites for this connector.
+
+## Obtaining Credentials
+There are no credentials needed for this connector.
+
+## Supported Operations
+### Get a quote
+Returns an array with one quote.
+### Get a number of quotes
+Returns an array with a count of quotes.
+### Search for a term
+Returns an array of quotes matching term without case sensitivity.
+
+## Known Issues and Limitations
+There are no known issues at this time.
```

Comment:
There is no Deployment Instruction section in the Readme.md, and it is a required section.
Please follow this [link ](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors)as an example.